dem.cc

```diff
@@ -0,0 +1,93 @@
+#include <iostream>
+#include <cmath>
```

Comment:
Do cincludes first.

Reply:
Do you mean moving cmath above iostream?

Reply:
Yes please